Images/.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  eslint:
+    name: ESLint
+    runs-on: ubuntu-22.04
```

Comment:
[Optional] You seem to have set up your project with the incorrect linters setup. Kindly note that you were supposed to use the [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) rather than [javascript](https://github.com/microverseinc/linters-config/blob/master/javascript) linters setup since this is a HTML, CSS and JavaScript project.